.hadolint.yaml

```diff
@@ -3,3 +3,4 @@ ignored:
   - DL3006
   - DL3008
   - DL3013
+  - DL3059
```

Comment:
We need to either update docs or simply ignore this rule just once:
https://github.com/jupyter/docker-stacks/blob/b6104850fa83d19555884a1443e38dababe7df7b/docs/contributing/lint.md#image-lint